src/lib/Form/BaseSubmitHandler.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformUser\Form;
+
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+class BaseSubmitHandler implements SubmitHandler
+{
+    public function handle(FormInterface $form, callable $handler): ?Response
+    {
+        $data = $form->getData();
+
+        if ($form->isValid()) {
+            $handler($data);
```

Comment:
Shouldn't $handler callback return `Response`? Otherwise it's gonna be always null.